script/update-content-versions-and-check-for-archived-slugs.ts

```diff
@@ -50,10 +44,21 @@ async function updateRedirections(_bookId: string, currentVersion: string, _newV
       throw error;
     });
 
+  const redirectsBookPath = path.resolve(redirectsPath, bookId + '.json');
+
+  try {
+    // Create file with empty array only if it does not exixts
+    fs.writeFileSync(redirectsBookPath, '[]', { encoding: 'utf8', flag: 'wx' });
+  } catch {
```

Comment:
this would be less lines of code if you just did an if statement with `existsSync`

Reply:
since you have an unconditional write at the end of this logic, you could also do a ternary assignment defaulting `redirects` to `[]` if the file doesn't exist, then it'll be created at the end